src/_includes/theme-toggle.njk

```diff
@@ -0,0 +1,14 @@
+<section class="l-footer__theme-switcher" aria-label="Theme switcher">
```

Comment:
Not specifically a change I am requesting, but seems like this `<section>` element and the parent `<div>` element has the same class. So maybe we want to just keep either the `<section>` or the `<div>` with the aria-label, instead of both? Feels a bit over-structured to me. Is there a specific reason why that might be problematic?